odc/stats/plugins/lc_level3.py

```diff
@@ -0,0 +1,54 @@
+"""
+Land Cover Level3 classification
+"""
+
+from typing import Tuple
+import xarray as xr
+from ._registry import StatsPluginInterface, register
+
+NODATA = 255
+
+
+class StatsLccsLevel3(StatsPluginInterface):
+    NAME = "ga_ls_lccs_level3"
+    SHORT_NAME = NAME
+    VERSION = "0.0.1"
+    PRODUCT_FAMILY = "lccs"
+
+    @property
+    def measurements(self) -> Tuple[str, ...]:
+        _measurements = ["level3_class"]
+        return _measurements
+
+    def reduce(self, xx: xr.Dataset) -> xr.Dataset:
+
+        l34_dss = xx.classes_l3_l4
+        urban_dss = xx.urban_classes
+        cultivated_dss = xx.cultivated_class
+
+        cultivated_mask = l34_dss == 110
+        l34_cultivated_masked = xr.where(cultivated_mask, cultivated_dss, l34_dss)
```

Comment:
cultivated is masked by the veg in input already, so it's just matter of merging valid data (`data != nodata`)